doc/source/exclude_patterns.md

```diff
@@ -0,0 +1,35 @@
+# Exclude Pattern Settings 
```

Comment:
I guess this file came back during a rebase?

Reply:
Oops! Fill fix it up and squash the commits